src/coffee/sync/utils/product.coffee

```diff
@@ -562,8 +562,11 @@ class ProductUtils extends BaseUtils
               return
             deltaValue = @getDeltaValue(value)
             unless deltaValue
-              deltaValue = value[0]
-              delete deltaValue.value
+              # Taken from https://github.com/commercetools/nodejs/blob/ab8edfb41bdc7d429f20554d3d8a45ef251228f8/packages/sync-actions/src/product-actions.js#L286
+              if (value[0] && value[0].name)
```

Comment:
I didn't want to differ from the linked resource as it's easier to compare like this.